pkg/topology/availability_zones_test.go

```diff
@@ -198,9 +192,9 @@ var _ = Describe("Availability Zones and Zones", func() {
 	}
 
 	assertGetNamespaceFolderAndRPMoIDSuccess := func() {
-		for i := 0; i < numberOfAvailabilityZones; i++ {
-			assertGetNamespaceFolderAndRPMoIDSuccessForAZ(fmt.Sprintf("az-%d", i))
-		}
+		// for i := 0; i < numberOfAvailabilityZones; i++ {
```

Comment:
Do you want to add a comment why this is commented out, or just remove the code?